packages/a11y-testing/src/definitions/Button/toggleButtonBehaviorDefinition.ts

```diff
@@ -4,11 +4,11 @@ import { buttonBehaviorDefinition } from './buttonBehaviorDefinition';
 
 export const toggleButtonBehaviorDefinition: Rule[] = [
   BehaviorRule.root()
-    .forProps({ active: 'true' })
+    .forProps({ active: 'true', checked: true })
```

Comment:
had to add `checked` value because 
Northastar button has prop `active` but react-button has prop `checked`